extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -366,7 +367,15 @@ export class Visibility {
       this.resources_.push(resource);
     });
 
-    // TODO: support "hidden" spec.
+    // Hidden trigger
+    if (!this.visibilityListenerRegistered_) {
+      this.viewer_.onVisibilityChanged(() => {
```

Comment:
Can we only register this when the first "hidden" visibility listener is registered?